File: day1/app/src/main/java/fan/akua/day1/MainActivity.java
package fan.akua.day1;

import android.app.Activity;
import android.os.Bundle;

import androidx.annotation.Nullable;

import fan.akua.day1.databinding.ActivityHelloBinding;

public class MainActivity extends Activity {
    protected ActivityHelloBinding binding;

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        binding = ActivityHelloBinding.inflate(getLayoutInflater());
        setContentView(binding.getRoot());
    }
}

File: day11/app/src/main/kotlin/fan/akua/day11/MainActivity.kt
package fan.akua.day11

import android.graphics.Color
import android.os.Build
import android.os.Bundle
import android.view.View
import androidx.activity.SystemBarStyle
import androidx.activity.enableEdgeToEdge
import androidx.appcompat.app.AppCompatActivity
import androidx.core.view.ViewCompat
import androidx.navigation.findNavController
import androidx.navigation.ui.setupWithNavController
import com.google.android.material.bottomnavigation.BottomNavigationView
import fan.akua.day11.databinding.ActivityMainBinding
import fan.akua.day11.ktx.akuaEdgeToEdge


class MainActivity : AppCompatActivity() {
    private lateinit var binding: ActivityMainBinding
    override fun onCreate(savedInstanceState: Bundle?) {
        akuaEdgeToEdge()
        super.onCreate(savedInstanceState)
        binding = ActivityMainBinding.inflate(layoutInflater)
        setContentView(binding.root)

        val navView: BottomNavigationView = binding.navView

        val navController = findNavController(R.id.nav_host_fragment_activity_main)

        navView.setupWithNavController(navController)
    }
}